source/common/res/features/popup-calculator/settings.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
My latest commit fixed the errors and reverts the popup-calculator settings file to the new style.